### ISSUE TITLE
Redet not working?

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1420]
I just tried to redetermine record ID 29075596.

The Redet comment box contains the predefined template text with its tokens, but there is no preview option to check how the tokens will appear - should there be?

I then saved the redet, but at that point I got a elastic search failure message. Following that, the record has not been redetermined, and so far it has not reappeared in the ES view either (it's now missing from the ES verification and Explore pages).

I then testing another redetermination, for record ID 28834987. This time there was no ES error message, but it seems that the redet has failed to be applied again, and the record is currently missing from the ES views on iRecord.

The lack of ES visibility may just be down to timing (I did the redets about 15-20 minutes ago), but if I check the records via the warehouse interface they have not been redetermined.

**Comment by johnvanbreda** [comment]
@kitenetter I've been unable to reproduce this on my local setup or on iRecord in training mode. Can you arrange a HAR file for me please, plus let me know the exact time it failed so I can check the Drupal and warehouse logs. Or, let me know a record you'd like redetermined and what to, plus a comment, so I can masquerade as you and recreate it.

**Comment by kitenetter** [comment]
Record IDs 28834987 and 29075596 have now reappeared in the ES explore page, but neither has been redetermined, nor have the comments that I added been saved.

I just tried to redetermine record ID 28834987 again, and again I got the "ES query failed" message, and again it has not been redetermined nor stored a comment. HAR file:

[irecord_2023_01_28.zip](https://github.com/BiologicalRecordsCentre/iRecord/files/10528347/irecord_2023_01_28.zip)

The above has all been done using the Brave browser. I will try in Edge as well once the record reappears in the ES verification page.
